TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -795,4 +796,30 @@ public object GetValue(object o, SymbolTable table)
             return null;
         }
     }
+
+
+    public class MaxArgsCountAttribute : Attribute
+    {
+        public object GetValue(object o, SymbolTable table)
+        {
+            var program = o as IProcCaller;
+            if (program != null)
+            {
+                int maxArgCount = 0;
+                Definitions definitions = (Definitions) new DefinitionsAttribute().GetValue(o, table);
+                foreach (var f in definitions.functions.Public)
+                {
+                    if (f is FunctionDeclaration)
+                    {
+                        int count = (f as FunctionDeclaration).Profile.Parameters.Count;
+                        if (count > maxArgCount) maxArgCount = count;
+                    }
+                }
+
+                return maxArgCount;
```

Comment:
This part can be simplifed using the LINQ method `Max`. Also if all the information is taken from the `Definitions` attribute, why not add the property in the `DefinitionsAttribute` class instead of creating a new one ?

Reply:
done: using Linq and deplace in DefinitionsAttribute.
https://github.com/TypeCobolTeam/TypeCobol/pull/1672/commits/8928c8c2c550d45632df004469afa36c55002b4c

Reply:
Code has been optimized.
https://github.com/TypeCobolTeam/TypeCobol/pull/1672/commits/8bdba8d7e369bdae22c7e2792441082b6c2ead2d